Ginger/GingerCoreNET/GeneralLib/Email.cs

```diff
@@ -38,23 +40,86 @@ public enum eEmailMethod
         [IsSerializedForLocalRepository]
         public int ID { get; set; }
 
+        private string mMailFrom;
         [IsSerializedForLocalRepository]
-        public string MailFrom { get; set; }
+        public string MailFrom
+        {
+            get
+            {
+                return mMailFrom;
+            }
+            set
+            {
+                if (mMailFrom != value)
+                {
+                    mMailFrom = value;
+                    OnPropertyChanged(nameof(MailFrom));
+                }
+            }
+        }
 
+        private string mMailTo;
         [IsSerializedForLocalRepository]
-        public string MailTo { get; set; }
+        public string MailTo
+        {
+            get
+            {
+                return mMailTo;
+            }
+            set
+            {
+                if (mMailTo != value)
+                {
+                    mMailTo = value;
+                    OnPropertyChanged(nameof(MailTo));
+                }
+            }
+        }
 
         [IsSerializedForLocalRepository]
```

Comment:
Why on property changed is needed for few fields and for few of them it is not needed?

Reply:
only for those we have ValueExpression added property changed.